### PR TITLE
Updated release notes and changelog.

### PR DESCRIPTION
**Pull request by mookerji**
/cc @mfine @denniszollo @fnoble

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,8 +6,37 @@ For a detailed index of changes, please see the
...
+v0.39 <a name="v0.39"></a>
+--------------------------
+
+Bug fixes, minor additions, and stability enhancements since
+[v0.33](#v0.33).
+
+#### Specifications
+
+- New external events trigger message.
+- The ephemeris message is now included in the list of observations to
```
Comment (fnoble) [inline RELEASE_NOTES.md]: "in the list of observations" -> "in the list of messages"

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -6,8 +6,37 @@ For a detailed index of changes, please see the
...
+  could only create message objects by deserializing from an existing
+  SBP message!
+- Robustness improvements for serializing message types to/from JSON.
+- Piksi serial link command line tool (`serial_link.py`) now lives in
+  [piksi_tools](https://github.com/swift-nav/piksi_tools/).
+
+#### Datasheet Documentation
+
+- More stable datasheet generation when adding new messages ("the
+  floats are too damn high!" and automatic versioning from git tags).
```
Comment (fnoble) [inline RELEASE_NOTES.md]: Maybe nix the "damn" as this is so public facing? :)

Comment (mookerji) [inline RELEASE_NOTES.md]: HAHAHAHHAHAHAHA

**Comment by mfine** [comment]
lgtm

**Pull request merged**